packages/gamut-labs/src/brand/ProLogoAlt/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { colors } from '@codecademy/gamut-styles';
+
+export type ProLogoAltProps = {
+  ariaLabel?: string;
+  backgroundColor?: string;
+  textColor?: string;
+  width?: number;
+};
+
+const backgroundDefault = colors.yellow;
+const textDefault = colors.black;
+const widthDefault = 50;
+
+export const ProLogoAlt: React.FC<ProLogoAltProps> = ({
+  ariaLabel = 'ProLogo',
```

Comment:
Nit: I think this'd be a little more informative / easy to read for screenreaders:

```suggestion
  ariaLabel = 'Codecademy Pro Logo',
```